readchar/_win_read.py

```diff
@@ -21,7 +21,7 @@ def readkey() -> str:
         raise KeyboardInterrupt
 
     # if it is a normal character:
-    if ch not in "\x00\xe0":
+    if ch not in "\x00\xc3\xa0":
```

Comment:
what is the reason for this change?

only `0x00` and `0xe0` are documented prefix chars that mark a multi-char sequence, so what are the other chars?

Reply:
The reason is that when reading with msvcrt.getwch(), \xe0 is read as \xc3\xa0.

Reply:
Oh, I see where the problem is. At first, I tried converting it to UTF-8 for detection(like `if ch.encode("utf-8") not in not in b"\x00\xc3\xa0":`), but then I reverted it to the original implementation (like `if ch not in "\x00\xe0":`). This should be changed back.